CONTRIBUTING.md

```diff
@@ -169,6 +169,22 @@ akka-cluster/scalafmtAll
 akka-persistence/scalafmtAll
 ```
 
+To format the Java source code:
+```
+sbt
+project akka-actor
+javafmtAll
+```
```

Comment:
Actually, [`sbt applyCodeStyle`](https://github.com/akka/akka/blob/master/build.sbt#L27) is a task that will handle headers, scalafmt and javafmt at once. Maybe we should promote that.

Reply:
I will update with that.

Reply:
@ignasi35 I have updated it.

Reply:
Brilliant, thanks!